src/main/webapp/assets/js/utils.js

```diff
@@ -98,6 +98,17 @@ var utilPopWindow = function(url, name){
     return false;
 };
 
+/*used by modal (other instructor field) */
+var validateUniqname = function (value) {
+  var value = $.trim(value)
+  var letterOnly = /^[a-zA-Z]+$/;  
```

Comment:
Shouldn't this regex (and the one that occurs later) check only for [a-z], since the uniqname is being converted to lowercase elsewhere?  If it does need to check for upper- and lowercase, I think the "i" flag could be used at the end of the regex, to make it shorter:

  /^[a-z]+$/i